contents/java/JUnit.md

```diff
@@ -0,0 +1,258 @@
+<frontmatter>
+  title: JUnit
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# JUnit
+
+<b>Author: [Lin Si Jie](https://github.com/sijie123)</b>
+
+Reviewers: [Aadyaa Maddi](https://github.com/amad-person), [Marvin Chin](https://github.com/marvinchin)
+
+## What is JUnit?
+_____
+
+JUnit is an open-source <trigger for="modal:automatedTesting2103" trigger="click">automated testing</trigger> framework for Java. Using its <popover effect="fade" content="Application Programming Interfaces">APIs</popover>, developers can write and run tests to check for correctness in their software engineering projects. 
```

Comment:
I feel like a popover might be better UX here compared to a modal, since it is placed closer to the term itself 🙂 Also, making it a click trigger made me a little uncertain about clicking it, because I thought it was a link elsewhere.